METADATA_EXAMPLES.md

```diff
@@ -0,0 +1,11 @@
+**Code.gov Metadata Schema 2.0.0 Examples**
+
+Code.gov developed this document to showcase good examples of metadata in current agency code.json files.  We hope it helps when creating your agency code.json file.  
+
+Code.gov is the nation’s catalog for U.S. Government source code.  The platform allows for discoverability of custom developed software that is reusable or open sourced.  The current metadata schema 2.0.0 allows agency inventory metadata to be displayed properly on Code.gov.
+
+Benefits of having good metadata:
```

Comment:
Just an FYI, this paragraph is probably not displaying as you intended:

![image](https://user-images.githubusercontent.com/1918027/43097667-c98d7076-8e8a-11e8-825b-e718e0b36da4.png)

I suggest adding a `*` before each enumerated item. 

Ex.

```
Benefits of having a good metadata:

* Allows for proper and accurate display of code inventory on Code.gov and agency websites.
```